server/testutils/factories/cas1PremisesSummary.ts

```diff
@@ -6,7 +6,7 @@ import { apAreaFactory } from './referenceData'
 
 export default Factory.define<Cas1PremisesSummary>(() => ({
   id: faker.string.uuid(),
-  name: `${faker.person}`,
```

Comment:
I noticed a bug here - the name of the premises was appearing as `[object:object]`

Reply:
Good catch, also weird to be using the `faker.person` for a premises summary factory